src/util/getDepth.ts

```diff
@@ -0,0 +1,18 @@
+import { hashContext } from '.'
+import { Context } from '../types'
+import { State } from './initialState'
+import { unroot } from './unroot'
+
+/** Get max depth of a visible context. */
+export const getDepth = (state: State, context: Context): number => {
+  const contextIndex = state.thoughts.contextIndex
+  const contextEncoded = hashContext(context)
+  const parentEntry = contextIndex[contextEncoded]
+  const children = parentEntry?.children ?? []
```

Comment:
I believe you can can use `getAllChildren` here to decouple from the implementation details of the `contextIndex`.

Reply:
This isn't the latest code and I'm using [getChildrenRanked](https://github.com/cybersemics/em/blob/5c44ef66c5a536c0002ea367aacb893da74937de/src/reducers/existingThoughtMove.ts#L70) there.
But `getAllChildren` makes more sense since we don't need ranks.